tests/Fuerte/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+add_executable(fuertetest
+  VstTest.cpp
+  ConnectionBasicTest.cpp
+  ConnectionConcurrentTest.cpp
+  ConnectionFailuresTest.cpp
+  ConnectionTimeoutsTest.cpp
+#    test_10000_writes.cpp
+  main.cpp
+)
+
+target_link_libraries(fuertetest
+    fuerte
+    gtest
+    boost_boost
+    arango
+    arango_v8server
+    arango_v8
+    arangoserver
+    ${MSVC_LIBS}
+    ${OPENSSL_LIBRARIES}
+    ${CMAKE_THREAD_LIBS_INIT}
+)
+
+target_include_directories(fuertetest SYSTEM PRIVATE
+  ${V8_INTERNAL_INCLUDE_DIR}
+  ${INCLUDE_DIRECTORIES}
+  ${CMAKE_SOURCE_DIR}/3rdParty/fuerte/src
```

Comment:
instead of this, couldn't you add in 3rdParty/fuerte/CMakeLists.txt
```
target_include_directories(fuerte PUBLIC src)
```
Then a library that links to fuerte has automatically this include path.